src/main/webapp/view/discussion/discussion.js

```diff
@@ -147,6 +154,30 @@ function updateNewCommentTimestamp(timeMs) {
   newCommentTimestampMs = timeMs;
 }
 
+/**
+ * Returns an array of the `DiscussionComment`s within
+ * `TIME_TOLERANCE_MILLISECONDS` milliseconds to the `timestampMilliseconds`.
+ *
+ * <p>This returns an empty array if no elements are nearby.
+ */
+function getNearbyDiscussionComments(timestampMilliseconds) {
+  const nearby = [];
+  // currentRootDiscussionComments is already sorted by timestamp.
+  for (const element of currentRootDiscussionComments) {
+    const commentTime = element.comment.timestampMilliseconds;
+    if (commentTime < timestampMilliseconds - TIME_TOLERANCE_MILLISECONDS) {
+      // Before the start of the range, continue to next.
+      continue;
+    }
+    if (commentTime > timestampMilliseconds + TIME_TOLERANCE_MILLISECONDS) {
+      // Outside of range, there will be no more.
+      break;
```

Comment:
Instead of break, you can return nearby.

Reply:
Done